modules/nw-multus-bond-cni-object.adoc

```diff
@@ -87,4 +96,41 @@ The following example configures a secondary network named `bond-net1`:
         "gateway": "10.56.217.1"
     }
 }
-----
\ No newline at end of file
+----
+
+The following example configures a secondary network named `bond-tlb-net` with the `xmitHashPolicy` feature enabled:
+
+[source,json]
+----
+{
+ "type": "bond",
+ "cniVersion": "0.3.1",
+ "name": "bond-tlb-net",
+ "mode": "tlb",         
+ "xmitHashPolicy": "layer2+3", <1>
+ "failOverMac": 0,         
+ "linksInContainer": true,
+ "miimon": "100",
+ "mtu": 1500,
+ "links": [
+       {"name": "net1"},
+       {"name": "net2"}
+   ],
+  "ipam": {
+        "type": "host-local",
+        "subnet": "10.57.218.0/24", 
+        "routes": [{
+        "dst": "0.0.0.0/0"
+        }],
+        "gateway": "10.57.218.1"
+    }
+}
+----
+
+<1> This parameter dictates how outgoing network traffic is distributed (load-balanced) across the active member interfaces `net1` and `net2` within the bond.
```

Comment:
Avoid using parentheses: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses
I don't think it changes the meaning so either use distributed as it's easier to translate.
I would also switch around net1 and net2 interfaces so you name the interfaces and then define what they are.
```suggestion
<1> This parameter dictates how outgoing network traffic is distributed across the `net1` and `net2` active member interfaces within the bond.
```